src/Analyzers/Core/Analyzers/xlf/AnalyzersResources.zh-Hans.xlf

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en" target-language="zh-Hans" original="../AnalyzersResources.resx">
+  <file datatype="xml" source-language="en" target-language="zh-HANS" original="../AnalyzersResources.resx">
```

Comment:
Is there a reason to prefer one capitalization over the other? or in the end, does it matter?

Reply:
No reason.  Either way should be fine.